transports/quic/src/crypto.rs

```diff
@@ -0,0 +1,150 @@
+use ed25519_dalek::{Keypair, PublicKey};
+use libp2p_core::PeerId;
+use quinn_proto::crypto::Session;
+use quinn_proto::TransportConfig;
+use std::sync::Arc;
+
+pub struct CryptoConfig<K> {
+    pub keypair: Keypair,
```

Comment:
For noise this should be ed25519::Keypair, however for tls this should be libp2p_core::identity::Keypair.

Reply:
fixed in https://github.com/dvc94ch/rust-libp2p/pull/13